src/solvers/sweep_plans/sweep_plans.jl

```diff
@@ -71,12 +73,13 @@ function forward_sweep(
   regions = collect(
     flatten(map(i -> forward_region(edges, i; region_kwargs, kwargs...), eachindex(edges)))
   )
-
   if reverse_step
     reverse_regions = collect(
       flatten(
         map(
-          i -> reverse_region(edges, i; region_kwargs=reverse_kwargs, kwargs...),
+          i -> reverse_region(
+            edges, i; reverse_edge=reverse_edges, region_kwargs=reverse_kwargs, kwargs...
```

Comment:
```suggestion
            edges, i; reverse_edge=reverse_edges, region_kwargs, kwargs...
```

Reply:
How come you wan't to not pass `reverse_kwargs` here? They seem like they are being passed (although they are made identical to `region_kwargs` by the look of the code in `tdvp_sweep_plan`)

Reply:
Oh, I see, I misread the code and thought it was `region_kwargs=region_kwargs` as opposed to `region_kwargs=reverse_kwargs`, never mind.